release/Magarena/scripts/The_Scorpion_God.groovy

```diff
@@ -0,0 +1,59 @@
+def DelayedTrigger = {
+    final MagicSource staleSource, final MagicPlayer stalePlayer, final MagicCard refCard ->
```

Comment:
refCard can't be used directly as it could be from a different game instance. It needs to be mapped into the current game instance, see how release/Magarena/scripts/Grave_Betrayal.groovy does it.

Reply:
Thanks. Pushed the fix.

Reply:
The MagicEvent construct still takes a `refCard`, that variable is no longer around. It should be `mappedCard`